dashboard/app/controllers/ai_lesson_summaries_controller.rb

```diff
@@ -0,0 +1,73 @@
+class AiLessonSummariesController < ApplicationController
+  before_action :authenticate_user!
+
+  # GET /ai_lesson_summaries/show?user_id=1&lesson_id=2
+  def show
+    @ai_lesson_summary = AiLessonSummary.find_by(
+      user_id: params[:user_id],
+      lesson_id: params[:lesson_id]
+    )
```

Comment:
Wondering if this could be a find_or_create so that a call to `show` will return a summary unless an error has occurred?

Reply:
Or will this not work because it needs to go into the active job queue?

Reply:
Discussed offline, not really a path we need to consider for now!